app/permission/rest.py

```diff
@@ -0,0 +1,56 @@
+
+from flask import (jsonify, request, abort, Blueprint, current_app)
+from app.schemas import (
+    permission_schema,
+    permission_schema_load_json)
+from app.errors import register_errors
+
+
+permission = Blueprint('permission', __name__)
+register_errors(permission)
+
+
+@permission.route('', methods=['GET'])
+def get_permissions():
+    data, errors = permission_schema.dump(
+        permission_schema.get_query(filter_by_dict=request.args), many=True)
+    if errors:
+        abort(500, errors)
+    return jsonify(data=data)
+
+
+@permission.route('/<permission_id>', methods=['GET'])
+def get_permission(permission_id):
+    inst = permission_schema.get_query(filter_by_dict={'id': permission_id}).first()
+    if not inst:
+        abort(404, 'Permission not found')
```

Comment:
Actually we want to start adding more details for logging, so this could read 'Permission not found for id: {}'.format(permission_id)